samples/quickstart.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START recaptcha_enterprise_quickstart]
+/**
+ * Instantiate the reCAPTCHA Enterprise client and make request.
+ * @param {string} projectNumber The project number to use when making requests.
+ */
+async function quickstart(projectNumber) {
+  // const projectNumber = [PROJECT_NUMBER];
+
+  // Create the reCAPTCHA client library.
+  const {
+    RecaptchaEnterpriseServiceV1Beta1Client,
+  } = require('@google-cloud/recaptcha-enterprise');
+  const client = new RecaptchaEnterpriseServiceV1Beta1Client();
+
+  // format the path to the project, projectNumber should be
+  // provided.
+  const formattedParent = client.projectPath(projectNumber);
```

Comment:
I ultimately couldn't get a simple integration test running, for a couple of reason's documented here:

https://github.com/googleapis/nodejs-recaptcha-enterprise/issues/12

I'd advocate landing this as a starting point, and we can come back and add more samples now that we have a foundation in place.